apps/api/prisma/migrations/20230509134331_add_document_latest_version_column/migration.sql

```diff
@@ -0,0 +1,19 @@
+BEGIN TRY
+
+BEGIN TRAN;
+
+-- AlterTable
+ALTER TABLE [dbo].[Document] ADD [latestVersionId] INT CONSTRAINT [Document_latestVersionId_df] DEFAULT 1;
```

Comment:
I note that all existing documents will have a NULL latestVersionId, and therefore the 2nd version gets numbered as 1.  Do we need to update all existing docs to have latestVersionId = 1 if they are NULL?

Reply:
The code adds version number 1 automatically for new documents after this, but existing docs in the database will still have a null value, won't they?  Existing docs with null versionid will need to be updated to "1"